editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -7555,12 +7626,14 @@ Node3DEditor::Node3DEditor(EditorNode *p_editor) {
 	p->add_separator();
 	p->add_check_shortcut(ED_SHORTCUT("spatial_editor/view_origin", TTR("View Origin")), MENU_VIEW_ORIGIN);
 	p->add_check_shortcut(ED_SHORTCUT("spatial_editor/view_grid", TTR("View Grid"), Key::NUMBERSIGN), MENU_VIEW_GRID);
+	p->add_check_shortcut(ED_SHORTCUT("spatial_editor/keep_gizmo_onscreen", TTR("Keep Gizmo On Screen")), MENU_KEEP_GIZMO_ONSCREEN);
```

Comment:
@Calinou Any idea for a default hotkey?

Reply:
<kbd>Ctrl + Alt + G</kbd> could be used, as it's not used for anything else currently.

Reply:
Added Ctrl+Alt+G.